infor-design/design-system
# Restructure tokens for ease of comparing and creating themes

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
As of the conversations we've been having with the way that Theo works in token conversion, versus the way we want to do themes, we realized it would be better to 
- simplify the structure of the tokens
- use a script to compare and make sure all themes have the same tokens
- removed a buggy package of `glob-fs` in favor of `glob`
- refactored some scripts for better errors/handling

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/design-system/issues/202

**Steps necessary to review your pull request (required)**:

- Pre-req:
    - `npm prune && npm i`

- Testing a good scenario:
  1. `npm run build:tokens:web`
  1. Enjoy the successful terminal output

- Testing a bad scenario:
  1. Change one of the theme files `aliases` or `props` property/key name.
    For example, in `design-tokens/theme-soho-dark/variables.alias.yml` change "TEXT_COLOR" to "CAT_COLOR"
      ```
      ...
      aliases:
        # Text Colors
        CAT_COLOR: "{!PALETTE_GRAPHITE_10}"
      ```
  1. `npm run build:tokens:web`
  1. See the console output showing details of what doesn't match.
